.travis.yml

```diff
@@ -37,6 +39,9 @@ before_install:
 before_script:
   - bundle exec rake db:create db:migrate
 
-bundler_args: --without development --deployment --retry=3 --jobs=3
+install:
+  - bash -c "if [ '$RAILS42' == '1' ]; then bundle update --retry=3 --jobs=3 rails rails-observers; fi"
+  - bash -c "if [ '$RAILS_MASTER' == '1' ]; then bundle update --retry=3 --jobs=3 arel rails rails-observers seed-fu; fi"
```

Comment:
This kinda sucks :(

I wonder if we should just be more strict in the Gemfile and do an unconditional `bundle update (all the things)`. But I guess this list is pretty small at the moment, so is it worth it? I'm not sure